examples/cpp/list_bucket.cpp

```diff
@@ -0,0 +1,15 @@
+#include "google/cloud/storage/client.h"
```

Comment:
Can you add this to CI? 

For that we need:

1. A script in `./ci`
2. A docker image where that script can run (probably something with cmake preinstalled?)
3. Then add it to the matrix here: https://github.com/fsouza/fake-gcs-server/blob/cf3fcb083e19553636419818e29f84825bd1e13c/.github/workflows/main.yml#L120-L139

Reply:
Oh 3 minutes isn't too bad given that CI runs things in parallel and running the tests on Windows takes 6 minutes anyway. We could definitely optimize it later though (would probably require maintaining a custom image, or leverage some caching mechanism in GitHub Actions).